lib/std/priority_queue.zig

```diff
@@ -86,6 +89,23 @@ pub fn PriorityQueue(comptime T: type) type {
             return first;
         }
 
+        pub const Error = error{BoundsError};
+
+        /// Remove and return element at index. If there aren't enough
+        ///  elements to remove from idx, returns null. Indices are in
+        ///  the same order as iterator, which is not necessarily
+        ///  priority order.
+        pub fn removeIdx(self: *Self, idx: usize) !T {
+            if (self.len <= idx)
+                return Self.Error.BoundsError;
```

Comment:
The docs says "returns null", but it's an error union. Also, should this be an assertion rather than an error?

Reply:
Whoops, caught the docs and code disagreeing. You're probably right about it being an assertion. It saves a branch, and `remove` doesn't return an error.